### PR TITLE
Feature/task 3411 UI tests for statistics

### PR DESCRIPTION
**Pull request by tcfos**
## Description
fixed broken tests
UITests for statistics cards

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4608

### CODE REVIEW

File: src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		// WHEN
+		app.setPreferredContentSizeCategory(accessibililty: .normal, size: .S)
+		app.launch()
+		app.swipeUp()
+
+		// THEN
+		XCTAssert(app.staticTexts[title1].waitForExistence(timeout: .short))
+		XCTAssert(app.staticTexts[title2].waitForExistence(timeout: .zero))
+		XCTAssert(app.staticTexts[title3].waitForExistence(timeout: .zero))
+		XCTAssert(app.staticTexts[title4].waitForExistence(timeout: .zero))
```
Comment (whiskey) [inline src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift:36]: why not…?
```suggestion
		XCTAssert(app.staticTexts[title2].exists)
		XCTAssert(app.staticTexts[title3].exists)
		XCTAssert(app.staticTexts[title4].exists)
```

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift [hunk 2]
```diff
@@ -305,5 +305,27 @@ enum AccessibilityIdentifiers {
...
+			
+			enum KeySubmissions {
+				static let title = "AppStrings.Statistics.Card.KeySubmissions.title"
+			}
+			
+			enum ReproductionNumber {
+				static let title = "AppStrings.Statistics.Card.ReproductionNumber.title"
+			}
+		}
+	}
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:329]: I personally don't see the need to be this specific for these cards.
```suggestion
        enum Statistics {
		static let Infections = "AppStrings.Statistics.Card.Infections.title"
		static let Incidence = "AppStrings.Statistics.Card.Incidence.title"
		static let KeySubmissions = "AppStrings.Statistics.Card.KeySubmissions.title"
		static let ReproductionNumber = "AppStrings.Statistics.Card.ReproductionNumber.title"
	}
```

File: src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift (hunk 1 above)
Comment (tcfos) [reply src/xcode/ENA/ENAUITests/Home/ENAUITests-Statistics.swift]: Sure. Fixed.

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift (hunk 2 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift]: fixed.

**Pull request merged**